projects/plugins/jetpack/extensions/blocks/subscriptions/deprecated/v6/save.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import { RawHTML } from '@wordpress/element';
+import {
+	getColorClassName,
+	__experimentalGetGradientClass as getGradientClass,
+	getFontSizeClass,
+} from '@wordpress/block-editor';
+import classnames from 'classnames';
+import { reduce } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import definedAttributes from '../v3/attributes';
+
+export const DEFAULT_BORDER_RADIUS_VALUE = 0;
+export const DEFAULT_BORDER_WEIGHT_VALUE = 1;
+export const DEFAULT_PADDING_VALUE = 15;
+export const DEFAULT_SPACING_VALUE = 10;
+export const DEFAULT_FONTSIZE_VALUE = 16;
```

Comment:
```suggestion
export const DEFAULT_FONTSIZE_VALUE = '16px';
```



I'm not sure where this version of `save.js` came from, but it appears to be missing a few of the changes you can see in the [copy on master currently](https://github.com/Automattic/jetpack/blob/d001afc08e7a70209cfdaafbbca14a91b0e65a25/projects/plugins/jetpack/extensions/blocks/subscriptions/save.js) (minus your `successMessage` changes of course). For example this `save.js` is missing the `buttonWidth` attribute.

Reply:
I have made the required changes.